app/views/users/index.html.erb

```diff
@@ -1,10 +1,30 @@
 <h1>Users</h1>
 
-<% @users.each do |user| %>
-  <p><%= link_to(user.given_name_safe, user_path(user)) %></p>
-<% end %>
+<table id="user-list-table">
+  <thead>
+    <tr class="user-list-header-row">
+      <th>Net ID</th>
+      <th>Given name</th>
+      <th>Family name</th>
+      <th>Full name</th>
+      <th>ORCID</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @users.each do |user| %>
+    <tr class="user-list-data-row">
+      <td>
+        <%= link_to(user.uid, user_path(user)) %>
+        <% if user.uid == current_user.uid %>
+          &nbsp;(you)
+        <% end %>
```

Comment:
This is why I want to have access to `current_user`. I want to indicate the user which one is _their_ record, in this case we display the label "(you)" right next to their name.